2.4/examples/replica/mongodb-clustered.json

```diff
@@ -119,6 +119,20 @@
               {
                 "name":  "mongodb",
                 "image": "openshift/mongodb-24-centos7",
+                "readinessProbe": {
+                  "timeoutSeconds": 1,
+                  "initialDelaySeconds": 3,
+                  "exec": {
+                    "command": [ "/bin/sh", "-i", "-c", "mongostat --host 127.0.0.1 -u admin -p $MONGODB_ADMIN_PASSWORD -n 1 --noheaders"]
```

Comment:
Since mongodb 2.6 mongo tools are developed in separate project. It is written in golang and it is quite large. mongo-tools package is not shipped in current images and with it image could be 80MB larger.

So I think it is better to use commands already installed in image. `mongo` shell,  `mongosniff` or `mongoperf`.